ui/raidboss/raidemulator.js

```diff
@@ -29,6 +29,7 @@ import defaultOptions from './raidboss_options';
 import '../../resources/defaults.css';
 import './raidemulator.css';
 
+// TODO: why is this IIFE, and why are these lets not just consts in the addEventListener?
```

Comment:
If I remember correctly, before the typescript conversion process and bundling, the IIFE was required to ensure that this executed after all the JS loaded, to allow the various classes to be available. Probably not required anymore?

Reply:
Yeah, I don't think it is.  I'll follow up after this and remove it.